influxdb_client/client/flux_table.py

```diff
@@ -95,3 +108,8 @@ def __str__(self):
         """Return formatted output."""
         cls_name = type(self).__name__
         return cls_name + "() table: " + str(self.table) + ", " + str(self.values)
+
+    def __repr__(self):
+        """Format for inspection."""
+        d = self.values
+        return f"<{type(self).__name__} field={d.get('_field')} value={d.get('_value')}>"
```

Comment:
Please use same format as for `FluxTable`:

```suggestion
        return f"<{type(self).__name__}: field={self.values.get('_field')}, value={self.values.get('_value')}>"
```